src/components/Asset/AssetContent/MetaFull.tsx

```diff
@@ -7,13 +7,14 @@ import { useWeb3 } from '@context/Web3'
 import { Asset, Datatoken, LoggerInstance } from '@oceanprotocol/lib'
 
 export default function MetaFull({ ddo }: { ddo: Asset }): ReactElement {
-  const [paymentCollector, setPaymentCollector] = useState<string>()
+  const [paymentCollector, setPaymentCollector] = React.useState<string>()
   const { isInPurgatory } = useAsset()
   const { web3 } = useWeb3()
 
   useEffect(() => {
     async function getInitialPaymentCollector() {
       try {
+        if (!web3 || !ddo) return
```

Comment:
so without wallet I can't see the payment collector?

Reply:
Good point, I've removed this